changelog.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## Unreleased
```

Comment:
I moved my previous changes in the README for changelog updates to the changelog file under a new `Unreleased` header since I'm not sure under what version you'd release this under, and it didn't feel right being presumptuous about the version being made in the README since I'm not the one producing a new package.